pkg/controller/kogitotrusty/kogitotrusty_controller.go

```diff
@@ -145,18 +144,3 @@ func (r *ReconcileKogitoTrusty) Reconcile(request reconcile.Request) (reconcile.
 
 	return reconcile.Result{}, nil
 }
-
-const (
-	// Collection of kafka topics that should be handled by the Trusty
-	kafkaTopicNameTraceEvents                 string = "kogito-tracing-decision"
-	kafkaTopicNameModelEvents                 string = "kogito-tracing-model"
-	kafkaTopicNameExplainabilityResultEvents  string = "trusty-explainability-result"
-	kafkaTopicNameExplainabilityRequestEvents string = "trusty-explainability-request"
-)
-
-var kafkaTopics = []services.KafkaTopicDefinition{
-	{TopicName: kafkaTopicNameTraceEvents, MessagingType: services.KafkaTopicIncoming},
-	{TopicName: kafkaTopicNameModelEvents, MessagingType: services.KafkaTopicIncoming},
-	{TopicName: kafkaTopicNameExplainabilityResultEvents, MessagingType: services.KafkaTopicIncoming},
-	{TopicName: kafkaTopicNameExplainabilityRequestEvents, MessagingType: services.KafkaTopicOutgoing},
-}
```

Comment:
Same thing about creating `KafkaTopics` if the `KogitoInfra` kind is from Strimzi operator.

Reply:
done